ddtrace/llmobs/_integrations/langchain.py

```diff
@@ -52,7 +52,7 @@ def llmobs_set_tags(
         span.set_tag_str(MODEL_NAME, span.get_tag(MODEL) or "")
         span.set_tag_str(MODEL_PROVIDER, model_provider or "")
 
-        self._llmobs_set_input_parameters(span, model_provider)
+        self._llmobs_set_input_metadata(span, model_provider)
```

Comment:
Might be worth renaming this to 

`self._llmobs_set_span_metadata`

Reply:
To clarify it's metadata for the whole span, not just the input

Reply:
Ahh good catch!